src/ts/components/subtitleoverlay.ts

```diff
@@ -86,6 +86,20 @@ export class SubtitleOverlay extends Container<SubtitleOverlayConfig> {
 
       this.applyConfToDom(labelToAdd.getDomElement());
 
+      // In this block all the logic for CEA-608 captions is applied
+      // It's based on the assertion that a cue with position metadata
+      // is CEA-608
+      if (event.position != null) {
+        let domElement = labelToAdd.getDomElement();
+        domElement.css('position', 'fixed');
```

Comment:
As discussed offline, please move all static styles into a stylesheet. Add a marker class `cea-608` that carries the styles, and add the marker when CEA-608 captions are detected (and remove it otherwise). An examples for adding a marker class can be found in `PlaybackTimeLabel`. Add the styling into `subtitlesettings/_subtitleoverlay-cea608.scss` and apply the marker directly to subtitle overlay (instead of each `SubtitleLabel`).